Library/test-helpers/Dockerfile.tenant

```diff
@@ -0,0 +1,9 @@
+FROM quay.io/centos/centos:stream9
+COPY id_*.pub /root/
+COPY lime_con_start.sh /usr/local/bin/lime_con_start
+RUN dnf install -y keylime-agent-rust util-linux-core openssh openssh-server which && \
+  dnf clean all && \
+  ssh-keygen -A && mkdir -p /root/.ssh && \
```

Comment:
My idea was that for tenant we would each time start a new container (i.e. the container won't be running as a server). From this perspective running ssh server there is probably not necessary.

Reply:
That also applies to lime_con_start script usage.

Reply:
DONE.